sources/utils/error_messages.c

```diff
@@ -6,13 +6,27 @@
 /*   By: qbeukelm <qbeukelm@student.42.fr>            +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2024/01/15 20:36:44 by quentinbeuk   #+#    #+#                 */
-/*   Updated: 2024/04/23 20:57:55 by quentinbeuk   ########   odam.nl         */
+/*   Updated: 2024/05/02 16:22:19 by qtrinh        ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "../../includes/error_messages.h"
 #include "../../includes/minishell.h"
 
+void	arg_error(void)
+{
+	g_exit_code = EXIT_FAILURE;
+	ft_putstr_fd("minishell: exit: too many arguments\n", STDERR_FILENO);
```

Comment:
Do not use ft_putstr_fd. Rather make a new error string and use the existing error messages.
In my opinion, the error messages don't have to be the same. I would rather just have one format.
`error string:` + `param`

Reply:
gotcha! i juts moved the code to the error messages part as i made these errors before our error message format was created ;)